modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -5358,6 +5358,23 @@ endif::[]
 
 // end::sasl_mechanisms[]
 
+=== schema_registry_enable_authorization
+
+Enables ACL-based authorization for Schema Registry requests. When `true`, Schema Registry
+uses ACL-based authorization instead of the default (`false`) `public/user/superuser` authorization model. Requires authentication to be enabled using the `schema_registry_api.authn_method`.
```

Comment:
Might be worth linking to the broker property `schema_registry_api.authn_method`.

Reply:
once our automation adds that property, I will link to it.